Code/GraphMol/MolStandardize/Charge.h

```diff
@@ -130,14 +133,17 @@ class RDKIT_MOLSTANDARDIZE_EXPORT Uncharger {
   Uncharger(bool canonicalOrdering) : Uncharger() {
     df_canonicalOrdering = canonicalOrdering;
   }
-  Uncharger(const Uncharger &other);
```

Comment:
Removing these makes sense, but please add a `default` declaration for them so that it's clear we made an intentional decision

Reply:
added.